examples/jenkins/pipeline/README.md

```diff
@@ -0,0 +1,59 @@
+This set of files will allow you to deploy a Jenkins server that is capable of executing Jenkins pipelines and
+utilize pods run on OpenShift as Jenkins slaves.
+
+To walk through the example:
+1) Stand up an openshift cluster from origin master, installing the standard imagestreams to the openshift namespace:
+
+$ oc cluster up
+
+If you do not use oc cluster up, ensure the imagestreams are registered in the openshift namespace, as well as the
+jenkins template represented by jenkinstemplate.json:
+
+oc create -f https://github.com/openshift/origin/blob/master/examples/image-streams/image-streams-centos7.json -n openshift
```

Comment:
URL should be `https://raw.githubusercontent.com/openshift/origin/master/examples/image-streams/image-streams-centos7.json` instead